css/style.css

```diff
@@ -1,19 +1,262 @@
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
+
+ /****** General styles *******/
+
+ body {
+
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    width: 100%;
+}
+header{
+    width: 90%;
+    margin-left: 20px;
+}
+
+/******   Navigation styles  *******/
+
```

Comment:
There's nothing wrong with using `float`, but try to think if you can solve the problem using only `flex` first.